packages/client/lib/MetadataClient.ts

```diff
@@ -15,22 +13,6 @@ import { getJson } from './functions';
 const debug = Debug('sphereon:openid4vci:metadata');
 
 export class MetadataClient {
-  /**
```

Comment:
Why are these methods gone and not replaced with similar functions. You want someone using the lower-level classes to be able to retrieve Server Metadata from the initiation/credential offers in multiple ways